lib/currency.rb

```diff
@@ -0,0 +1,29 @@
+module Currency
+  extend self
+  
+  def args(*currency)
+    currency = currency.flatten.compact
+    return if currency.blank?
```

Comment:
Inacceptable. `blank?` is rails-only goodness. Why would you restrict usage of this module to rails only?